### ISSUE TITLE
Standardized on D versus d_t for Dataflow Fact

### ISSUE THREAD

**Issue by blipper** [secure-software-engineering/phasar#415]
Some places use D and some use d_t.  We should just pick some to standardize on the template type name.

**Comment by MMory** [comment]
I agree on that. We intend to go with d_t in the future. This might be a low-hanging fruit for newbies :) I just labelled this as good first issue.

**Comment by fabianbs96** [comment]
Is this still an issue? We added an entry to our [coding guidelines](https://github.com/secure-software-engineering/phasar/wiki/Coding-Conventions#type-aliases) were this is specified.
